config/application.yml.default

```diff
@@ -427,6 +427,7 @@ development:
   show_unsupported_passkey_platform_authentication_setup: true
   sign_in_recaptcha_score_threshold: 0.3
   skip_encryption_allowed_list: '["urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost"]'
+  socure_webhook_enabled: false
```

Comment:
I think this needs to be defined at the top-level scope for the build to pass. I think we only need to define it once as `false` everywhere, unless we're ready to enable it for development?